src/core/comm/coll.cc

```diff
@@ -506,5 +515,5 @@ extern "C" {
 
 void legate_cpucoll_finalize(void) { legate::comm::coll::collFinalize(); }
 
-void legate_cpucoll_initcomm(int id) { legate::comm::coll::collInitComm(id); }
+int legate_cpucoll_initcomm(void) { return legate::comm::coll::collInitComm(); }
```

Comment:
again, let's not use `void` in the signature for a function that takes no argument.